apps/emqx_resource/src/schema/emqx_resource_schema.erl

```diff
@@ -76,46 +76,30 @@ worker_pool_size(default) -> ?WORKER_POOL_SIZE;
 worker_pool_size(required) -> false;
 worker_pool_size(_) -> undefined.
 
-resume_interval(type) -> emqx_schema:duration_ms();
+resume_interval(type) -> emqx_schema:timeout_duration_ms();
 resume_interval(importance) -> ?IMPORTANCE_HIDDEN;
 resume_interval(desc) -> ?DESC("resume_interval");
 resume_interval(required) -> false;
 resume_interval(_) -> undefined.
 
-metrics_flush_interval(type) -> emqx_schema:duration_ms();
+metrics_flush_interval(type) -> emqx_schema:timeout_duration_ms();
 metrics_flush_interval(importance) -> ?IMPORTANCE_HIDDEN;
 metrics_flush_interval(required) -> false;
 metrics_flush_interval(_) -> undefined.
 
-health_check_interval(type) -> emqx_schema:duration_ms();
+health_check_interval(type) -> emqx_schema:timeout_duration_ms();
 health_check_interval(desc) -> ?DESC("health_check_interval");
 health_check_interval(default) -> ?HEALTHCHECK_INTERVAL_RAW;
 health_check_interval(required) -> false;
-health_check_interval(validator) -> fun health_check_interval_range/1;
 health_check_interval(_) -> undefined.
 
-health_check_interval_range(HealthCheckInterval) when
```

Comment:
this should be kept ?
max is `3_600_000`

Reply:
For timeouts in receive blocks, it's `4294967295` ms, but we may restrict it further for this field.

I'll add it back with a test.